requirements.lock

```diff
@@ -4,6 +4,8 @@ astroid==2.2.5
 atomicwrites==1.3.0
```

Comment:
I think that after this PR, or maybe I will have a chance to do it during this PR to separate requirements into requirements which are necessary for proper server work and for testing purposes because this file is too big and too much unnecessary requirements is installed to virutal enviroment on server.